src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2673,7 +2684,21 @@ private void EmitDefaultValue(TypeSymbol type, bool used, SyntaxNode syntaxNode)
                     }
                 }
 
-                EmitInitObj(type, true, syntaxNode);
+                if (type.IsPointerType() || type.SpecialType == SpecialType.System_UIntPtr)
```

Comment:
I didn't understand how this change relates to the PR. Was it covered in tests with VerifyIL?